### PR TITLE
fix: object status link focus

### PR DESCRIPTION
**Pull request by Betrozov**
## Related Issue
Closes https://github.com/SAP/fundamental-styles/issues/2009
Part of https://github.com/SAP/fundamental-ngx/issues/3414

## Description
Added focus for Object status as a link.

### Before:
Screenshot at 14-38-09

### After:
Screenshot at 14-38-48

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [X] All values are in `rem`
- [X] Text elements follow the truncation rules
- [X] hover state of the element follow design spec
- [X] focus state of the element follow design spec
- [X] active state of the element follow design spec
- [X] selected state of the element follow design spec
- [X] selected hover state of the element follow design spec
- [X] pressed state of the element follow design spec
- [X] Responsiveness rules - the component has modifier classes for all breakpoints
- [X] Includes Compact/Cosy/Tablet design
- [X] RTL support
2. The code follows fundamental-styles code standards and style
- [X] only one top level `fd-*` class is used in the file
- [X] BEM naming convention is used
- [X] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [X] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [X] `fd-reset()` mixin is applied to all elements
- [X] Variables are used, if some value is used more than twice.
- [X] Checked if current components can be reused, instead of having new code.
3. Testing
- [X] tested Storybook examples with "CSS Resources" `normalize` option 
- [X] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [X] Verified all styles in IE11
- [X] Updated tests
4. Documentation
- [X] Storybook documentation has been created/updated
- [X] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

**Review by olshansky — APPROVED** [review]

File: src/object-status.scss [hunk 1]
```diff
@@ -291,6 +291,8 @@ $inverted-color-accents: (
 
   // CLICKABLE OBJECT STATUS
   &--link {
+    @include fd-fiori-focus-within(0);
```
Comment (JKMarkowski) [inline src/object-status.scss:294]: Is there any reason of using focus within, instead of basic focus?

Comment (JKMarkowski) [reply src/object-status.scss:294]: Could you also add `0.0625rem`, instead of 0? Just to follow focus visual concept from fiori 3

Comment (Betrozov) [reply src/object-status.scss]: forgot to change,
@JKMarkowski  thanks
changed to fd-fiori-focus

Comment (Betrozov) [reply src/object-status.scss]: @JKMarkowski fixed

**Review by InnaAtanasova — APPROVED** [review]

**Review by stefanoScalzo — APPROVED** [review]
LGTM

**Pull request merged**